README.md

```diff
@@ -82,24 +82,20 @@ A Yarn Workspace maintaining a single version across all packages.
 
 #### System Requirements
 
-- Docker `17.12.0`+
-- Docker Compose
+- [nvm](https://github.com/nvm-sh/nvm)
+- [yarn](https://classic.yarnpkg.com/lang/en/docs/install)
 
 #### Install and Build
 
 ```console
+nmv install && \
```

Comment:
```suggestion
nvm install && \
```

Reply:
Where's the face-palm reaction when you need it :stuck_out_tongue: